core/eolearn/tests/test_utils/test_common.py

```diff
@@ -50,3 +53,40 @@ def test_is_discrete_type(number_type, is_discrete):
         numpy_dtype = np.dtype(number_type)
 
     assert is_discrete_type(numpy_dtype) is is_discrete
+
+
+@pytest.mark.parametrize(
+    "map_function, data, spatial_axes, expected",
+    [
+        (partial(np.resize, new_shape=(2, 2)), np.zeros(shape=0), (1, 2), np.zeros((2, 2))),
+        (partial(np.resize, new_shape=(2, 2)), np.ones((2, 3, 4, 1)), (1, 2), np.ones((2, 2, 2, 1))),
+        (partial(np.resize, new_shape=(3, 2)), np.ones((2, 3, 4, 1)), (0, 1), np.ones((3, 2, 4, 1))),
+        (partial(np.resize, new_shape=(2, 3)), np.ones((2, 3, 4, 1)), (0, 2), np.ones((2, 3, 3, 1))),
+        (partial(np.resize, new_shape=(1, 2)), np.ones((2, 3, 4, 1)), (2, 3), np.ones((2, 3, 1, 2))),
```

Comment:
many of these are either redundant or hard to understand.
use a `new_shape` that is clearly different, such as `5, 6`.

Reply:
not axis, `new shape`

Reply:
test case updated

Reply:
Updated in some cases, still want to test shrinking.